config/olm/metrics.service.yaml

```diff
@@ -5,17 +5,16 @@ metadata:
   name: addon-operator-metrics
   namespace: addon-operator
   labels:
-    app: addon-operator-metrics-service
+    app.kubernetes.io/name: addon-operator
   # In production, the following annotation will make service-ca-operator to create a Secret called `metrics-server-cert` on the fly and manage its lifecycle
   annotations:
     service.beta.openshift.io/serving-cert-secret-name: metrics-server-cert
 spec:
+  type: ClusterIP
+  sessionAffinity: None
   ports:
     - port: 8443
       name: https
       targetPort: 8443
-    - port: 8080
-      name: http  ## [only for dev/test purposes]
-      targetPort: 8080
```

Comment:
Is it safe to remove these? We might want to use them during integration tests to query the metrics endpoint? @thetechnick

Reply:
we use files from https://github.com/openshift/addon-operator/tree/main/config/deploy for testing purpose

Reply:
The same file also ends up in the production OLM bundle, ideally there would be no difference between the bundle we test in OpenShift CI and what we deploy via app-interface.

Ref: https://github.com/openshift/addon-operator/blob/main/magefile.go#L312

Reply:
The `ServiceMonitor` in `config/deploy/monitoring/servicemonitor.yaml` that we deploy in the development environment (where we don't use the bundle), is still referencing the http port, so we should probably keep that for now.

Reply:
okay, i will add it back for consistency